### ISSUE TITLE
Fedora 34 Server - Ventoy - Error In Installation Media

### ISSUE THREAD

**Issue by jdb130496** [ventoy/Ventoy#888]
I tried ventoy USB with EFI boot system to install latest FEDORA 34 Server edition on one of my Pocket USB drive (1.5 TB). I booted the laptop with ventoy - it went fine - I selected FEDORA 34 ISO and then usual messages of Anaconda, appeared and then the first GUI screen of installtion. But on that very first installation screen it showed "installation media not found" message on that button, I then clicked on installation media button and went inside and selected ISO on the Ventoy USB - it failed with unknown error - Finally I had to select ISO from another USB (Not the Ventoy one) media attached to laptop with Fedora 34 ISO on that and then everything went fine and installation media and base repository was detected from that media - Does that mean the Ventoy yet to support Fedora 34 server on it's latest 1.0.43 version? 

Thank you in advance for support and efforts. 

I have not attached any image of errors, as I went ahead with the installation from other USB media skipping ISO on ventoy USB.

In short, I could boot the FEDORA 34 Server ISO with Ventoy USB but could go ahead with complete installation. For installation ISO detection, I had to rely on other USB.

Please look into or help me if I have done something wrong.

**Comment by ventoy** [comment]
Does F34 WorkStation edition work? Does F33 Server edition  work?

**Comment by jdb130496** [comment]
I have not tried F33 Server Edition or F34 Workstation. So will try and then post.

**Comment by ventoy** [comment]
It's a bug, I have fixed it.
Please use the following CI release for test:
https://github.com/ventoy/Ventoy/actions/runs/827259842

**Comment by jdb130496** [comment]
Thank you. Tested and found working with the bug fixed in the link given above. Hopefully also working with F34 Workstation - I have not tested that. Closing.

**Comment by ventoy** [comment]
F34 Wokstation has been tested with success.
